src/libraries/System.Formats.Asn1/src/System.Formats.Asn1.csproj

```diff
@@ -0,0 +1,61 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
+    <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
+    <NoWarn>$(NoWarn);CS1574;CS3016;CA5379;CA5384</NoWarn>
```

Comment:
We don't do it always but in general is good to add a comment above NoWarn explaining what does each mean and why do we want to disable it. I see that two of them are code analysis ones which is probably fine, but it would be good to understand the other two.